examples/storage/couchdb/couchdb-petset.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: "apps/v1alpha1"
+kind: PetSet
+metadata:
+  name: couchdb001
+  namespace: default
+spec:
+  serviceName: couchdb001-backend
+  replicas: 3
+  template:
+    metadata:
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+        pod.alpha.kubernetes.io/init-containers: '[
+            {
+            "name": "configer-couch",
+            "image": "glyndwr/kube-init-couchdb",
+            "command": ["/bin/bash", "-c", "./configer_couch.sh"],
```

Comment:
Yes it looks like @James1912 corrected the spelling in 5c68009 and forgot to update it here. Good catch.